app/logic/getUpcomingEvents.py

```diff
@@ -0,0 +1,27 @@
+from app.models.program import Program
+from app.models.interest import Interest
+from app.models.event import Event
+from app.models.user import User
+from datetime import date, datetime, time
+
+def getUpcomingEventsForUser(user):
+
+    interestedEvent = (Event.select(Event, Program, Interest)
+                            .join(Program)
+                            .join(Interest, on=(Event.program == Interest.program))
+                            .where(Interest.user == user)
+                            )
+    upcomingEvents = []
+    for event in interestedEvent.objects():
+        today = date.today()
+        times = datetime.now().time()

```

Comment:
`times` isn't a very clear variable name. Like the comment above, I'd either move it out of the loop (with a different name, like `currentTime`), or just use it directly in the conditional.

Reply:
Done!